src/Meshes/Builders/capsuleBuilder.ts

```diff
@@ -64,182 +62,177 @@ VertexData.CreateCapsule = function(
             )
         ).length();
 
-    // Total length for v texture coord
+        // Total length for v texture coord
     var vl = radiusTop * alpha
                 + cone_length
                 + radiusBottom * (Math.PI / 2 - alpha);
 
-    // generate vertices, normals and uvs
-    var v = 0;
-    for (y = 0; y <= capsTopSegments; y++) {
-
-        var indexRow = [];
-
-        var a = Math.PI / 2 - alpha * (y / capsTopSegments);
-
-        v += radiusTop * alpha / capsTopSegments;
-
-        var cosA = Math.cos(a);
-        var sinA = Math.sin(a);
-
-        // calculate the radius of the current row
-        var _radius = cosA * radiusTop;
-
-        for (x = 0; x <= radialSegments; x ++) {
-
-            var u = x / radialSegments;
-
-            var theta = u * thetaLength + thetaStart;
-
-            var sinTheta = Math.sin(theta);
-            var cosTheta = Math.cos(theta);
-
-            // vertex
-            vertex.x = _radius * sinTheta;
-            vertex.y = halfHeight + sinA * radiusTop;
-            vertex.z = _radius * cosTheta;
-            vertices.push(vertex.x, vertex.y, vertex.z);
-
-            // normal
-            normal.set(cosA * sinTheta, sinA, cosA * cosTheta);
-            normals.push(normal.x, normal.y, normal.z);
-            // uv
-            uvs.push(u, 1 - v / vl);
-            // save index of vertex in respective row
-            indexRow.push(index);
-            // increase index
-            index ++;
+        //var groupCount = 0;

```

Comment:
this was removed as well. why reintroducing it?

Reply:
Something has to be broken with my setup, I never reintroduced this  and everything should have matched your updates.  Really getting frustrated with my system, this never happened on my laptop.

Reply:
And it's even worse when trying to do gitlab projects.

Reply:
also it takes 45 mins to a hour for me to run a build it's like pulling teeth.  I do not think it's supposed to take that long.

Reply:
omg no..more around 5 to 10

Reply:
That's why when ever I say "Ill get right to it" then it takes me a hour and half...  I get the coding done right away, then I sit and have to wait for days...

Reply:
Whoa the plot thickens, this line of code is not even on my local....

we are talking about the //var groupCount =0; correct?

Reply:
yes I removed all comments yesterday

Reply:
wtf then, this is nutty.  I dont even have these comments in my code on my local but this is what it pushed?!?!?!

asdjk.fhasjkldfhjkasdfjklh Ok deep breath...  I am so sorry about this.  I will resolve it now.  I am started a build as soon as this conversation started and I am waiting for it to wrap up now.

Reply:
My build crashed VS code... starting it over.  I will commit as soon as that finished and will post the log to see if we can notice something from that.